examples/cassandra/cassandra-controller.yaml

```diff
@@ -26,7 +26,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          image: gcr.io/google_containers/cassandra:v6
+          image: gcr.io/google_containers/cassandra:v7
```

Comment:
Do we want to commit the jar file too?
examples/cassandra/image/kubernetes-cassandra.jar

To be honest, I'm not sure that's the best idea...